src/pepr/policies/security.ts

```diff
@@ -25,14 +25,38 @@ import { exemptionAnnotationPrefix, isExempt, markExemption } from "./exemptions
  */
 When(a.Pod)
   .IsCreatedOrUpdated()
-  .Mutate(markExemption(Policy.DisallowPrivileged))
+  .Mutate(request => {
+    markExemption(Policy.DisallowPrivileged)(request);
+    if (request.HasAnnotation(`${exemptionAnnotationPrefix}.${Policy.DisallowPrivileged}`)) {
+      return;
+    }
+
+    // Check if any containers defined in the pod do not have the `allowPrivilegeEscalation` field present. If not, include it and set to false.
+    for (const container of containers(request)) {
+      container.securityContext = container.securityContext || {};
+      if (
+        container.securityContext.allowPrivilegeEscalation === undefined &&
+        //Skip mutation if privileged
+        !container.securityContext.privileged &&
+        !container.securityContext.capabilities?.add?.includes("CAP_SYS_ADMIN")
+      ) {
+        container.securityContext.allowPrivilegeEscalation = false;
+      }
+    }
+    annotateMutation(request, Policy.DisallowPrivileged);
```

Comment:
nit: the placement of `annotateMutation` could mean that something gets annotated even though it didn't pass the conditional for any of its containers. We should probably have some flag `wasMutated` that is only set to true inside of that conditional, and check that before annotating?

Reply:
Good catch! What if we put the call to `annotateMutation` inside the conditional? Right below line 43. What that achieve the same effect?

Reply:
I think that may work, didn't dive into the code to validate that calling it multiple times would be okay here though (since multiple containers could pass that check and result in calling that func 2x, 3x, etc on a single pod).

Reply:
Fair point. In my testing, I have found that the `annotateMutation` function will add a policy name to the value of the annotation even if that value already exists in the list.  I will implement your suggestion and call `annotateMutation` the way you suggested 👍.